boards/arm/96b_argonkey/96b_argonkey.dts

```diff
@@ -70,6 +70,11 @@
 
 &i2s5 {
 	status = "ok";
+
+	mp34dt05 {
+		compatible = "st,mp34dt05";
```

Comment:
Provide yaml file matching this compatible.
It should be instanciated as a i2s-device, which rely on a i2s bus (should be somehow similar to spi / spi-device model).
Then, you'll get a DT_MP34DT05_BUS_NAME or simialr you'll be able to use in the driver

Reply:
I think I did. See 7939a51. The system generates infact:

#define DT_ST_MP34DT05_40015000_MP34DT05_LABEL             "MP34DT05"
#define DT_MP34DT05_DEV_NAME       DT_ST_MP34DT05_40015000_MP34DT05_LABEL

Isn't it correct?

Reply:
@avisconti ,

> I think I did

Indeed, i overlooked that part, my bad.
You then need to establish this is a device of the i2s bus, cf next comment